stripy/cartesian_meshes.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, extent, spacingX, spacingY, refinement_levels=0, tree=False):
 
         xy = np.vstack([xy0, xy])
 
-        super(square_border, self).__init__(xy[:,0], xy[:,1], permute=False, tree=tree)
+        super(square_border, self).__init__(xy[:,0], xy[:,1], permute=True, tree=tree)
```

Comment:
How can this be that we need to switch on `permute` here? I manually ensured that the first 3 points are not collinear...

Reply:
I agree - that was my attempt to refine the bug which I am not seeing on native installs. It seems to appear at odd times on different platforms.